fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,13 +1,14 @@
-With EventYay Attendee, you can browse through different events happening nearby or all over the world. You can get detailed information about the events and can even buy event tickets.
-The app also offers other cool features such as similar events and easy check-in using QR code. If that’s not enough, the app also allows you to favorite different events and all of them can be accessed from favorite section.
-
-Eventyay Attendee App provides features for users like:-
+With EventYay Attendee, you can browse through different events happening nearby or anywhere in the world. You can get detailed information about the events and buy tickets. The app also offers other features such as displaying ticket details and easy check-in using QR code. If that’s not enough, the app also allows you to favorite events and search events by their name.
```

Comment:
`Eventyay`. `Y` shouldn't be capital.

Reply:
I think it should be, Event+yay, here event description is started and the exact name is with the app.

Reply:
But the app name is Eventyay and not EventYay

Reply:
Can you please check out the tag here: https://blog.fossasia.org/tag/eventyay/

Reply:
Ok